### PR TITLE
LPS-171590, LPS-171597, LPS-171628, LPS-171629, LPS-171630, LPS-171631, LPS-171632 | GIVI | Test Automation | Master

### PR DESCRIPTION
**Pull request by andreefarias**
https://issues.liferay.com/browse/LPS-171590
https://issues.liferay.com/browse/LPS-171597
https://issues.liferay.com/browse/LPS-171628
https://issues.liferay.com/browse/LPS-171629
https://issues.liferay.com/browse/LPS-171630
https://issues.liferay.com/browse/LPS-171631
https://issues.liferay.com/browse/LPS-171632

### CODE REVIEW

**Comment by andreefarias** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomLayouts.testcase [hunk 1]
```diff
@@ -96,24 +99,218 @@ definition {
...
+
+		task ("Then the new entry is saved on the Relationship Tab Parent") {
 			AssertElementPresent(
-				key_tabName = "Relationship Tab",
-				locator1 = "ObjectAdmin#ANY_TAB_ON_ENTRY");
+				key_fieldLabel = "Entry Test B",
+				locator1 = "ObjectAdmin#VIEW_OBJECT_FIELD_LABEL");
+		}
+
+		task ("And the entry is present on Relationship Tab Child") {
```
Comment (epark18) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomLayouts.testcase:211]: Condense this into the "Then" block. It is usually ideal to have just one block for your assertions

**Review by epark18 — COMMENTED** [review]
Great job on the revisions! There's just a couple of small changes, and then this should be good to go.

**Comment by andreefarias** [comment]
Thanks! I followed your recommendation.

**Comment by epark18** [comment]
Great job on these!

**Comment by epark18** [comment]
ci:test:sf

**Comment by epark18** [comment]
ci:forward

**Pull request closed without merging**